upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,43 @@
 == Migration Changes
 
+=== Migrating to 13.0.0
+
+==== Upgrade to Wildfly 23
+
+The {project_name} server was upgraded to use Wildfly 23 as the underlying container. This does not directly involve any
+specific {project_name} server functionality, however, note these changes related to migration:
+
+Dependency updates::
+  The dependencies were updated to the versions used by the Wildfly 23 server. For example, Infinispan is now `11.0.9.Final`.
+
+Configuration changes::
+  A few configuration changes exist in the `standalone(-ha).xml` and `domain.xml` files. You should follow the <<_install_new_version>>
+  section to handle the migration of configuration files automatically. However, here are the most important changes, which you may need
+  if you made your own configuration changes:
+
+  * The `module` attribute of Infinispan cache containers is now *deprecated* (unused) and is *replaced with* the `modules` attribute,
+  representing the set of modules associated with this cache container's configuration. Moreover, there were also additional
+  changes to attributes of various elements, originating from the use of Wildfly 23 as the underlying container. For example,
+  the `managed-executor-service` and `managed-scheduled-executor-service` elements now recognize the new `hung-task-termination-period`
+  attribute. See link:https://docs.wildfly.org/23/wildscribe/index.html[the Wildfly 23 full model reference] for details.
+
+==== Upgrade to Wildfly 22
```

Comment:
Why do we need to have WF22 upgrade here? I've assumed we've skipped WF22 and upgraded from WF21 to WF23.

Reply:
@vramik Thanks for the review. I would say it's there just for our future tracking purposes (some incompatible features / changes were introduced in Wildfly 22 already, so if they would be listed under Wildfly 23 entry, they are still split to prevent the future digging / search for the release that introduced them won't be longer / confused by trying to identify those changes in Wildfly 23 tag).

This also corresponds / aligns with the way e.g. Wildfly Core is handling their component upgrades (listing multiple ones separately even in the same release). For example consider the release notes for [_15.0.0.Beta1_ tag](https://github.com/wildfly/wildfly-core/releases/tag/15.0.0.Beta1):

* ...
* _[WFCORE-5282] - Upgrade WildFly Elytron to 1.14.2.Final_ 
* _[WFCORE-5290] - Upgrade WildFly Elytron to 1.15.0.CR1_
* ...

And could find more such instances in Wildfly / Wildfly Core releases, if interested.

So this mirrors their approach. But if you want the Wildfly 22 entry to be removed / merged with Widlfly 23 one, I can squeeze it under one. Let me know.

@hmlnarik @martin-kanis @pdrozd @pskopek -^ What's your take on this?

Thanks,
Jan

Reply:
Either way works for me, so we can keep it as is

Reply:
@vramik @hmlnarik Thanks!